runtime/lua/vim/lsp/util.lua

```diff
@@ -1894,7 +1894,7 @@ function M.make_position_params(window, position_encoding)
   local buf = api.nvim_win_get_buf(window)
   if position_encoding == nil then
     vim.notify_once(
-      'warning: position_encoding is required, using the offset_encoding from the first client',
+      'warning: position_encoding param is required in vim.lsp.util.make_position_params, using the offset_encoding from the first client',
```

Comment:
the sentence is long enough that the comma is confusing, it sounds like it's referring to the first part of the sentence. Also we should speak of "position encoding", not `offset_encoding`

```suggestion
      'warning: position_encoding param is required in vim.lsp.util.make_position_params. Using the position encoding from the first client',
```